x-pack/plugins/integration_assistant/server/routes/register_routes.ts

```diff
@@ -13,12 +13,14 @@ import { registerRelatedRoutes } from './related_routes';
 import { registerPipelineRoutes } from './pipeline_routes';
 import type { IntegrationAssistantRouteHandlerContext } from '../plugin';
 import { registerAnalyzeLogsRoutes } from './analyze_logs_routes';
+import { registerCelInputRoutes } from './cel_routes';
 
 export function registerRoutes(router: IRouter<IntegrationAssistantRouteHandlerContext>) {
   registerAnalyzeLogsRoutes(router);
   registerEcsRoutes(router);
   registerIntegrationBuilderRoutes(router);
   registerCategorizationRoutes(router);
   registerRelatedRoutes(router);
+  registerCelInputRoutes(router);
```

Comment:
It would be better not to register the route if the experimental flag is disabled